indico/modules/events/templates/forms/session_block_form.html

```diff
@@ -0,0 +1,16 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
+
+{%- block content %}
+    {{ form_header(form) }}
+    {% call form_fieldset(_("Session properties"), collapsible=true, initially_collapsed=false) %}
+        {{ form_rows(form, fields=('session_title',)) }}
```

Comment:
I would add a property to the form that returns a list of those fields based on their name (`startswith('block_')` etc)